### ISSUE TITLE
Is there a replacement for setup_cache.sh?

### ISSUE THREAD

**Issue by hq6** [envoyproxy/envoy#34497]
**If you are reporting *any* crash or *any* potential security issue, *do not*
open an issue in this repo. Please report the issue via emailing
envoy-security@googlegroups.com where the issue will be triaged appropriately.**

*Title*: *Is there a replacement for setup_cache.sh?*

*Description*:
This [PR](https://github.com/envoyproxy/envoy/pull/30261) removed `setup_cache.sh` without any explanation of why it was being removed in either the PR description or the commit message.
Block was using this script to set up a remote bazel cache, so when the change was pulled in, CI times went from < 10 minutes to > 90 minutes.
Thus, I have two questions:
1. Why was the script removed? (If there were correctness issues, that would be good to now about.)
2. Is there a recommended alternative way to setup remote bazel caches if using Envoy's CI scripts, or are all downstream forks expected to implement their own?

**Comment by hq6** [comment]
@phlax Thanks for the quick response!
I think the examples you linked are sufficient. 
I'm closing the issue because the original question has been addressed.

The main problem was not knowing where to look, since I am not particularly familiar with bazel.
I think if we cross-link this issue to your original PR, it would fix the discoverability problem that I encountered.

Additionally, searching for bazel remote cache in any of the top-level MD files turned up nothing, but I'm not sure what the best place is to document this kind of thing.
```
ack -i 'bazel remote cache' *.md
```
